File: dataset_reduzido/codenet/Java/Code/s264878779.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String s = sc.next();
		int n = s.length();
		long ans = 0;
		for(int mask = 0 ; mask < 1<<(n - 1) ; mask++) {
			long sum = 0;
			String x = "" + s.charAt(0);
			for(int i = 0 ; i < n - 1 ; i++) {
				if((mask&(1<<i)) >= 1) {
					x += "+" + s.charAt(i + 1);
				} else {
					x += s.charAt(i + 1);
				}
			}
			String[] split = x.split("\\+");
			for(int i = 0 ; i < split.length ; i++) {
				sum += Long.valueOf(split[i]);
			}
			ans += sum;
		}
		System.out.println(ans);
	}
}

File: output/Llama-2-100/codenet/Java/Python/s264878779.py
import sys

def main():
    sc = input()
    n = len(sc)
    ans = 0
    for mask in range(1<<(n-1)):
        sum = 0
        x = sc[0]
        for i in range(n-1):
            if mask&(1<<i) > 0:
                x += "+" + sc[i+1]
            else:
                x += sc[i+1]
        split = x.split("+")
        for i in range(len(split)):
            sum += int(split[i])
        ans += sum
    print(ans)

